core/state_processor_rip7560.go

```diff
@@ -571,3 +528,27 @@ func validateValidityTimeRange(time uint64, validAfter uint64, validUntil uint64
 	}
 	return nil
 }
+
+func (epc *EntryPointCall) OnEnter(depth int, typ byte, from common.Address, to common.Address, input []byte, gas uint64, value *big.Int) {
+	if epc.OnEnterSuper != nil {
+		epc.OnEnterSuper(depth, typ, from, to, input, gas, value)
+	}
+	isRip7560EntryPoint := to.Cmp(AA_ENTRY_POINT) == 0
+	if !isRip7560EntryPoint {
+		return
+	}
+
+	if depth != 1 {
+		println("ONENTER WITH WRONG DEPTH!")
+		epc.err = errors.New("same")
+		return
+	}
+	if epc.Input != nil {
+		println("repeated call to ep callback")
```

Comment:
1. remove println
2. don't error with "same"
3. repeated call to validateAccount/validatePaymaster ("ep callback" is an internal term for us, not for users)